command/helper/config.go

```diff
@@ -23,6 +23,7 @@ type Config struct {
 	BlockGasTarget string                 `json:"block_gas_target"`
 	GRPCAddr       string                 `json:"rpc_addr"`
 	JSONRPCAddr    string                 `json:"jsonrpc_addr"`
+	METRICSAddr    string                 `json:"metrics_addr"`
```

Comment:
I would also create a type Telemetry and include Prometheus there because you will likely have more things on the future related to telemetry (like OpenTracing).
```
type Telemetry struct {
   Prometheus string
}
```

Reply:
@ferranbt Thanks for your input, will create a new type called telemetry.

Reply:
Commit [Link](https://github.com/0xPolygon/polygon-sdk/pull/216/commits/23a9953c3222ebfd6de9d46123e3e478e89bb7c4)